docs/source/configuration/tracing.mdx

```diff
@@ -81,6 +81,9 @@ telemetry:
       # https://zipkin.io/ (compliant with opentracing)
       zipkin: false
 
+      # https://aws.amazon.com/xray/ (compliant with opentracing)
```

Comment:
> (compliant with opentracing)

I'm actually curious what this comment means (even on the other options).  Is it _compatible_?  What makes it so?

Reply:
I _think_ that It means that the format is compatible with the OpenTelemetry OTLP exporter/collector.

Reply:
"OpenTelemetry" makes sense, but I was more curious about "OpenTracing" (which is different, as I understand it)